pages/api/account/[id]/forms.ts

```diff
@@ -0,0 +1,33 @@
+import { AccessControlError, createAbility } from "@lib/privileges";
+import { middleware, cors, sessionExists } from "@lib/middleware";
+import type { NextApiRequest, NextApiResponse } from "next";
+import { MiddlewareProps, WithRequired } from "@lib/types";
+import { getUser } from "@lib/users";
+import { getAllTemplatesForUser } from "@lib/templates";
+
+const getPublishedForms = async (
+  req: NextApiRequest,
+  res: NextApiResponse,
+  props: MiddlewareProps
+) => {
+  try {
+    const { session } = props as WithRequired<MiddlewareProps, "session">;
+    const ability = createAbility(session);
+
+    const accountId = req.query.id as string;
```

Comment:
Instead of force typing, I know it's a pattern we've done elsewhere, we can also do a quick type check that throws an error instead.

```suggestion
    if (Array.isArray(req.query.id) || !req.query.id) {
      return res.status(400).json({ error: "Bad request" });
    }
    const accountId = req.query.id;
```